File: Data Structures & Algorithms/two-integer-sum/submission-2.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        m = {}
        for i, value in enumerate(nums):
            if value in m:
                return [m[value], i] 
            else: 
                m[target - value] = i
        return []

File: Data Structures & Algorithms/missing-number/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    missingNumber(nums) {
        let res = 0
        for (let num in nums) {
            res ^= num
            res ^= nums[num]
        }
        return res ^ (nums.length)
    }
}
